elytron-oidc-client/src/main/resources/org/wildfly/extension/elytron/oidc/LocalDescriptions.properties

```diff
@@ -141,7 +141,8 @@ elytron-oidc-client.secure-deployment.socket-timeout-millis=The timeout for the
 elytron-oidc-client.secure-deployment.connection-ttl-millis=The amount of time in milliseconds for the connection to be kept alive. A value less than or equal to zero is interpreted as an infinite value.
 elytron-oidc-client.secure-deployment.connection-timeout-millis=The timeout for establishing a connection with the remote host in milliseconds. A timeout value of zero is interpreted as an infinite timeout, and a negative value is interpreted as undefined.
 elytron-oidc-client.secure-deployment.token-signature-algorithm=The token signature algorithm used by the OpenID provider.
-
+elytron-oidc-client.secure-deployment.credential.scope=The scope values are used to request specific or additional permissions for OIDC.
+elytron-oidc-client.secure-deployment.scope=The scope values are used to request specific or additional permissions for OIDC.
```

Comment:
Maybe we could reword this to something like:

A space separated list of scope values that should be used in the OIDC authentication request. Scopes can be used to request that specific sets of information be made available as Claim Values.